ovector.h

```diff
@@ -0,0 +1,35 @@
+#ifndef OVECTOR_H
+#define OVECTOR_H
+
+typedef struct OVector{
+    unsigned int size;
+    unsigned int capacity;
+    int* array;
+} OVector;
+
+
+int* createArray(int capacity);
+
+void initVector(OVector* vector);
+
+void deleteArray(OVector* vector);
+
+void resizeArray(OVector* vector, int newCapacity);
```

Comment:
please don't provide your internal implementation details to user API.

you can simply remove this function from header and forward declare it in the top of `.c` file